examples/platform/efr32/EFR32DeviceAttestationCreds.cpp

```diff
@@ -22,20 +22,24 @@
 
 #include "efr32_creds.h"
 #include "psa/crypto.h"
-#include "sl_token_api.h"
+
+extern uint32_t linker_nvm_end;
```

Comment:
This is good but could we create new variable in the ldscript. it could be directly CredBaseAddr and we would place it, right after the nvm3 section?